atlp-rwanda/alpha-ec-be
# feat(2FA for sellers)

#### What does this PR do?
this is 2FA (2Factor authentication) specifically for sellers.

#### Description of Task to be completed?
- before proceed to the user account there will be checking if user is a  seller 
- if not then proceed
- if  yes you will be asked to verify OTP sent to your email in order to proceed

#### How should this be manually tested?

- use login if you are seller to get otp on your email through api/users/login
- after verify your otp to proceed through api/users/verify/:token

#### What are the relevant pivotal tracker/Trello stories?

- id of the project
  #187300150

Responses 

[when seller logs in the response looks like this ](https://github.com/atlp-rwanda/alpha-ec-be/blob/ft-2fa-for-sellers-%23187300150/test/assets/response%20for%20login%20seller.png)

[when seller verify OTP they get this response](https://github.com/atlp-rwanda/alpha-ec-be/blob/ft-2fa-for-sellers-%23187300150/test/assets/login%20seller%20successfull.png)
